dev/io.openliberty.jakarta.pages.tld.3.0/bnd.bnd

```diff
@@ -0,0 +1,42 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.2
+-nouses=true
+
+Bundle-SymbolicName: io.openliberty.jakarta.pages.tld.3.0.javaee
+
+-pom: artifactid=io.openliberty.jakarta.pages.tld.3.0
+
+jakartaeeMe: true
+jakartaFinalJarName: io.openliberty.jakarta.pages.tld.3.0.jar
+jakartaFinalBundleId: io.openliberty.jakarta.pages.tld.3.0
+
+# Only publish the Jakarta one
+publish.wlp.jar.include: io.openliberty.jakarta.pages.tld.3.0.jar
+
+Export-Package: \
+    org.apache.taglibs.standard.tag.common.core;version="1.0.0", \
+    org.apache.taglibs.standard.tag.common.xml;version="1.0.0", \
+    org.apache.taglibs.standard.tag.el.core;version="1.0.0", \
+    org.apache.taglibs.standard.tag.el.fmt;version="1.0.0", \
+    org.apache.taglibs.standard.tag.el.sql;version="1.0.0", \
+    org.apache.taglibs.standard.tag.el.xml;version="1.0.0", \
+    org.apache.taglibs.standard.tag.rt.core;version="1.0.0", \
+    org.apache.taglibs.standard.tag.rt.fmt;version="1.0.0", \
+    org.apache.taglibs.standard.tag.rt.sql;version="1.0.0", \
+    org.apache.taglibs.standard.tag.rt.xml;version="1.0.0"
+
+-includeresource: @${repo;com.ibm.ws.javax.j2ee:jsp.tld.standard;1.2}
```

Comment:
We're thinking of switching to the glassfish impl.  Could you change this to: 

`-includeresource: @${repo;org.glassfish.web:jakarta.servlet.jsp.jstl;2.0}`

Reply:
We'll do this in another PR.